ec2listimg

```diff
@@ -29,160 +29,210 @@ from ec2imgutils.ec2imgutilsExceptions import (
+        metavar='AMI_ID'
+    )
+    publish_image_condition_group.add_argument(
+        '--image-name',
+        dest='imageName',
+        help='The image name of the image to be published (Optional)',
+        metavar='IMAGE_NAME'
+    )
+    help_msg = 'An image name fragment to match the image name of the image '
+    help_msg += 'to be published (Optional)'
+    publish_image_condition_group.add_argument(
+        '--image-name-frag',
+        dest='imageNameFrag',
+        help=help_msg,
+        metavar='IMAGE_NAME_FRAGMENT'
+    )
+    help_msg = 'A regular expression to match the image name of the image '
+    help_msg += 'to be published (Optional)'
+    publish_image_condition_group.add_argument(
+        '--image-name-match',
+        dest='imageNameMatch',
+        help=help_msg,
+        metavar='REGEX'
+    )
+    help_msg = 'Comma separated list of regions for publishing, all '
+    help_msg += 'integrated regions if not given (Optional)'
+    parser.add_argument(
+        '-r', '--regions',
+        dest='regions',
+        help=help_msg,
+        metavar='EC2_REGIONS'
+    )
+    parser.add_argument(
+        '-s', '--secret-key',
+        dest='secretKey',
+        help='AWS secret access key (Optional)',
+        metavar='AWS_SECRET_KEY'
+    )
+    help_msg = 'Set vebosety level 0 (default) image name, 1 image name and '
+    help_msg += 'ID, 2 full description'
+    parser.add_argument(
+        '--verbose',
+        type=int,
+        default=0,
+        help=help_msg,
+        dest='verbose'
+    )
+    parser.add_argument(
+        '--version',
+        action='version',
+        version=utils.get_version(),
+        help='Program version',
+    )
+
+    parsed_args = parser.parse_args(args)
+    return parsed_args
+
+
+# ----------------------------------------------------------------------------
+def get_config(args, logger):
+    """Function to get the configutation parsed from the configuration file
+    provided as an argument (or the default path)"""
+    config_file = os.path.expanduser(args.configFilePath)
+    config = None
+    if not os.path.isfile(config_file):
+        print('Configuration file "%s" not found.' % config_file)
+        sys.exit(1)
     try:
-        access_key = utils.get_from_config(args.account_name,
-                                           config,
-                                           None,
-                                           'access_key_id',
-                                           '--access-id')
-    except EC2AccountException as e:
-        print(format(e), file=sys.stderr)
+        config = utils.get_config(config_file)
+    except Exception as e:
+        logger.exception(e)
+        sys.exit(1)
+    return config
+
+
+# ----------------------------------------------------------------------------
+def get_access_key(args, config, logger):
+    """Function to get the access_key to AWS with the data provided as command
+    line args and config files.
```

Comment:
s/and config files/or configuration/

Stating "config files" is misleading as the function expects a config object.